localization/en-us.lua

```diff
@@ -396,6 +396,13 @@ return {
 					"discarded cards",
 				},
 			},
+			bl_cry_greed = {
+				name = "The Greed",
+				text = {
+					"+#1# blind requirement",
+					"per $5 when selected",
```

Comment:
Change to support the newly added localization vars in the main joker code. 
```suggestion
					"+#2# blind requirement",
					"per $#1# when selected",
```